content/events/one-pager.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Modern Infrastructure as Code
+type: page
+layout: event-one-pager
```

Comment:
We chatted here also and agreed we'd just name these all `show` for now, just for consistency, and then going forward consider how we'll handle multiple "show"s happening at the same time, etc.

Reply:
Ah, missed this comment when I was reviewing a minute ago. Naming all `show` for now SGTM.